builtInFunctions/migrateDataTrie.go

```diff
@@ -77,6 +90,42 @@ func (mdt *migrateDataTrie) ProcessBuiltinFunction(
 	return vmOutput, nil
 }
 
+func (mdt *migrateDataTrie) getAccountForMigration(callerAddr []byte, address []byte) (vmcommon.UserAccountHandler, error) {
+	if vmcommon.IsSystemAccountAddress(address) {
+		systemSCAccount, err := mdt.loadAccount(vmcommon.SystemAccountAddress)
+		if err != nil {
+			return nil, err
+		}
+
+		return systemSCAccount, nil
+	}
+
```

Comment:
please add an exception for the 0000....000 address, as that one will be considered intra-shard address by the shard coordinator

Reply:
Done.